src/HW14/Person.java

```diff
@@ -0,0 +1,25 @@
+package HW14;
+
+import java.util.Scanner;
+
+public class Person {
+    public String personName;
+    public String personsecondName;
+    public String city;
+    public int phonenumber;
+
+    public void personinfo () {
```

Comment:
this method should only print info, without read from scanner + method name should be personInfo()